source/bson-corpus/tests/timestamp.json

```diff
@@ -18,6 +18,11 @@
             "description": "Timestamp with high-order bit set on both seconds and increment",
             "canonical_bson": "10000000116100FFFFFFFFFFFFFFFF00",
             "canonical_extjson": "{\"a\" : {\"$timestamp\" : {\"t\" : 4294967295, \"i\" :  4294967295} } }"
+        },
+        {
+            "description": "Timestamp with high-order bit set: (4000000000, 4000000000)",
```

Comment:
Let's rename this to "Timestamp with high-order bit set on both seconds and increment (not UINT32_MAX)", since it's very much like the preceding test apart from the value being tested.

Reply:
Done!